ffi/jpeg.lua

```diff
@@ -95,9 +95,8 @@ function Jpeg.encodeToFile(filename, source_ptr, w, stride, h, quality, color_ty
     end
 
     turbojpeg.tj3Destroy(handle)
-    if jpeg_image[0] ~= nil then
-        turbojpeg.tj3Free(jpeg_image[0])
-    end
+    -- Handles NULL pointers sanely
+    turbojpeg.tj3Free(jpeg_image[0])
```

Comment:
This change isn't dangerous?

Reply:
Nope, the comment was supposed to help avoid similar questions in the future ^^.

```
 * @param buffer address of the buffer to free.  If the address is NULL, then
 * this function has no effect.
```

c.f., https://rawcdn.githack.com/libjpeg-turbo/libjpeg-turbo/main/doc/html/group___turbo_j_p_e_g.html#gaddb84fb6c81769e9faa0f5a63b296606